### DISCUSSION TITLE
Windows Task Sched Autorun

### DISCUSSION THREAD

**Discussion by Paradoxum** [mathoudebine/turing-smart-screen-python#262] [Q&A]
Somehow I was able to get it to run and not leave a tray icon, but I forgot to export the task and I just reinstalled windows and I can't seem to get it running again with no systray icon. What am I missing please?

Also nice just saw the 5" support post.
Thanks

**Comment by bado6100** [comment]
Is there a way to run this program auto in Win 11? I tried almost everything but no success till now.

**Reply by c0ldJS** [reply]
See [the documentation](https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-%3A-how-to-start#run-system-monitor-at-startup).

**Comment by brandonhiga** [comment]
@Paradoxum best to run it as Administrator - in the Task Properties, General, tick "Run with highest privileges" near the bottom.  Run it on-demand to test paths and permissions, and set the Trigger to At User Logon, or whatever trigger you want.  I'll post another comment on getting it to run on-demand at plug-in if you don't not keeping it plugged in all the time.
